docs/source/user_interface.rst

```diff
@@ -0,0 +1,54 @@
+==============
+User Interface
+==============
+
+Action Bar
+----------
+
+Across the top of the Packetry window is the Action Bar used to control Packetry.
+
+.. image:: ../images/action-bar.png
+  :alt: Action Bar
+
+- **Open Button**: Loads a ``.pcap`` file with the ``LINKTYPE_USB_2_0`` link-layer header type.
+- **Save Button**: Saves a ``.pcap`` file with the ``LINKTYPE_USB_2_0`` link-layer header type.
+- **Scan Button**: Searches for a capture device such as a Cynthion running analyzer gateware.
+- **Capture Button**: Starts capturing USB data with the connected capture device.
+- **Stop Button**: Stops capturing USB data.
+- **Device Drop-down**: Selects a capture device. If you don't see a device listed here, try using the Scan button.
+- **Speed Drop-down**: Selects the target device speed.
+
+
+Traffic Window
```

Comment:
I would call this the "Traffic View" - it's not a separate window.

Reply:
I had wondered about this. I took the name "Window" from the source code, but I was thinking something like "View", "Panel", "Pane", or "Area" might be better. Might "View" conflict with whatever we call it when we provide one or more additional ways to display traffic (interleaved tree or strict chronological packets)?

Reply:
It has that name in the source code only because it's a `Window` internally to GTK. It's not a window in terms of how it's presented to the user as a UI element.

I agree that "View" isn't the best choice, because we're going to offer different views within that same space.

I think "Pane" is the right choice. That matches what GTK calls those draggably-divided areas created by [Gtk.Paned](https://docs.gtk.org/gtk4/class.Paned.html).

Reply:
Okay, I'll go with "Pane". I had a look through documentation of some popular open source GUI applications (Inkscape, Gimp, LibreOffice) and saw "Pane" used once or twice.